content/blog/2019/06/2019-06-25-phase-1-multibranch-pipeline-support-for-gitlab.adoc

```diff
@@ -0,0 +1,324 @@
+
+=== Goals of this project
+
+1. Implement a lightweight GitLab Plugin that depends on GitLab API Plugin.
+
+2. Follow convention of 3 separate plugins i.e. `GitLab Plugin`, `GitLab API Plugin`, `GitLab Branch Source Plugin`.
+
+3. Implement GitLab Branch Source Plugin with support for Multi-branch Pipeline Jobs.
+
+4. Support new Jenkins features such as
+link:(https://github.com/jenkinsci/configuration-as-code-plugin)[Jenkins Code as Configuration] (JCasC),
+link:(https://github.com/jenkinsci/incrementals-tools/)[Incremental Tools].
+
+5. Clear & Efficient design.
+
+6. Support new SCM Trait APIs.
+
+7. Support Java 8 and above.
+
+== Building the plugin
+
+No binaries are available for this plugin as the plugin is in the very early alpha stage, and not ready for the general
+public quite yet.  If you want to jump in early, you can try building it yourself from source.
+
+=== Installation:
+
+1) Checkout out source code to your local machine:
+
+    [source, bash]
+    ---
+    git clone https://github.com/baymac/gitlab-branch-source-plugin.git
+
+    cd gitlab-branch-source-plugin
+    ---
+
+2) Install the plugin:
+    
+    [source, bash]
+    ---
+    mvn clean install 
+
+    or
+
+    mvn clean install -DskipTests # to skip tests
+    ---
+
+3) Run the Plugin:
+
+    [source, bash]
+    ---
+    mvn hpi:run # runs a Jenkins instance at localhost:8080
+    
+    or
+    
+    mvn hpi:run -Djetty.port={port} # to run on your desired port number 
+    ---
+
+    If you want to test it with your Jenkins server, after `mvn clean install` follow these steps in your Jenkins instance:
+    
+    1. Select `Manage Jenkins`
+    
+    2. Select `Manage Plugins`
+    
+    3. Select `Advanced` tab
+    
+    3. In `Upload Plugin` section, select `Choose file`
+    
+    4. Select `${root_dir}/target/gitlab-branch-source.hpi`
+    
+    5. Select `Upload` 
+    
+    6. Select `Install without restart`
+    
+== Usage
+
+Assuming plugin installation has done been already.
+
+=== Setting up GitLab Server Configuration on Jenkins
+
+1. On jenkins, select `Manage Jenkins`
+
+2. Select `Configure System`
+
+3. Scroll down to find the `GitLab` section
+
+   image::/images/post-images/2019-06-25-multibranch-pipeline-support-for-gitlab/gitlab-section.png[gitlab-section]
+
+4. Select `Add GitLab Server` | Select `GitLab Server`
+
+5. Now you will now see the GitLab Server Configuration options
+   
+   image::/images/post-images/2019-06-25-multibranch-pipeline-support-for-gitlab/gitlab-server.png[gitlab-server]
+   
+   There are 4 fields that needs to be configured:
+       
+    i. `Name` - Plugin automatically generates an unique server name for you. User may want to configure this field
+    to suit their needs but should make sure it is sufficiently unique. We recommend to keep it as it is.
+       
+    ii. `Server URL` - Contains the URL to your GitLab Server. By default it is set to "https://gitlab.com". User can
```

Comment:
Take a look at the preview around this point - https://github.com/jenkins-infra/jenkins.io/blob/bb8ca4018596e825d4e21ab4452891c399f5e358/content/blog/2019/06/2019-06-25-phase-1-multibranch-pipeline-support-for-gitlab.adoc 

Something goes sideways.  I think using `..` list notation instead of `ii` will fix this, but I'm not sure.